x/subspaces/types/models_feegrant_test.go

```diff
@@ -165,3 +166,50 @@ func TestGrant_Validate(t *testing.T) {
 		})
 	}
 }
+
+func TestGrant_GetExpiration(t *testing.T) {
```

Comment:
Can we add a test case with an allowance that does *not* have an expiration to make sure the method does not panic?

Reply:
Added test

Reply:
I meant having a test for a grant with no expiration time. That means a grant that will never expire (`ExpiresAt` that returns `nil`). I don't think we need to test the case in which `ExpiresAt` returns an error hoestly

Reply:
Ah, I see, apologize the misunderstanding.

Added the case which is `no expiration returns nil`.